### DISCUSSION TITLE
Unable to run zenoh example

### DISCUSSION THREAD

**Discussion by Just-secularized-monk** [eclipse-zenoh/roadmap#127] [Q&A]
<img width="545" alt="Unable to run zenoh example" src="https://github.com/eclipse-zenoh/roadmap/assets/37646573/5cb25d15-e5c9-495f-8731-951b53754b6d">
The problem I encountered is in the attach file. How to solve it? Thank you so much

**Comment by YuanYuYuan** [comment]
Hi @Just-secularized-monk, you could use `cargo run --example z_sub` to build and execute _z_sub.rs_.

**Reply by Just-secularized-monk** [reply]
<img width="612" alt="Why are the latency results not displayed on the console" src="https://github.com/eclipse-zenoh/roadmap/assets/37646573/3120ce7c-4b6e-47eb-8347-887f517434ad">
Hello, I have another question, I run z_pong and z_ping, why is the latency result not shown on the console

**Reply by YuanYuYuan** [reply]
Hi @Just-secularized-monk , I've replied to this question here https://github.com/eclipse-zenoh/roadmap/discussions/128#discussioncomment-8849668

Do you mind closing this thread and marking it solved?
